### PR TITLE
doc: zigbee: improve NCP sample docs

### PR DESCRIPTION
**Pull request by greg-fer**
Added setup overview image.
Linked NCP Host information to the sample.

Signed-off-by: Grzegorz Ferenc <Grzegorz.Ferenc@nordicsemi.no>

### CODE REVIEW

**Comment by greg-fer** [comment]
@edmont , you're on this PR for the one small edit to the FEM support file.

**Review by tomchy — APPROVED** [review]

**Review by edmont — APPROVED** [review]
> @edmont , you're on this PR for the one small edit to the FEM support file.

That change looks good to me.

**Comment by rlubos** [comment]
@carlescufi The checkpatch scanned the `svg` file for whatever reson and reports error, shall `*.svg` filetype be added to the checkpatch exclude list?

File: samples/zigbee/ncp/README.rst [hunk 1]
```diff
@@ -66,7 +72,7 @@ The communication channel uses Zephyr's :ref:`zephyr:uart_api` API.
 This serial device is selected with :option:`CONFIG_ZIGBEE_UART_DEVICE_NAME`.
 
 By default, the NCP sample communicates through the UART serialization (``UART0``).
-As a result, Zephyr's logger is configured to use ``UART1``, which is available through GPIO pins (``P1.00`` and ``P1.01``).
+As a result, Zephyr's logger is configured to use ``UART1``, which is available through GPIO pins (**P1.00** and **P1.01**).
 
 The ``UART0`` pins are configured by Devicetree :file:`overlay` files for each supported development kit in the :file:`boards` directory.
```
Comment (tomchy) [inline samples/zigbee/ncp/README.rst:78]: Please consider adding the following sentence in the section below:
```
This overlay file is not applied automatically, if the target board is set to the `nrf42840dongle_nrf52840`.
As a result, if you want to use the USB as the NCP communication channel when using the dongle, build it with the following command:

.. code-block:: console

   west build samples/zigbee/ncp -b nrf52840dongle_nrf52840 -- -DCONF_FILE='prj_usb.conf'
```

**Review by tomchy — APPROVED** [review]

**Comment by carlescufi** [comment]
> @carlescufi The checkpatch scanned the `svg` file for whatever reson and reports error, shall `*.svg` filetype be added to the checkpatch exclude list?

I will force-merge this.

**Pull request merged**